hbp-service-client/.gitignore

```diff
@@ -0,0 +1,61 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
`hbp-service-client` should not be included in the pull request, it should be installed using pip